2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+> handsomely. So handsomely in fact, that Kim can pay a student
+> assistant for a full year from the money.
+
+Our credibility as researchers in society hinges on the notion of proper
+research conduct, also known as “good research practice”. The
+digitalization of research has introduced complex digital research
+outputs, such as software and data sets. Although first
+recommendations (Katerbow and Feulner 2018) and policies (Scheliga et
+al. 2019) exist, they are far from being widely adapted. It is still
+somewhat unclear how to translate good research practice into research
+software practice, for example in terms of validity and reproducibility,
+but also pertaining to the responsible use of resources.
+
+While it is obvious that software qualifies as a potentially re-usable
+digital artifact, the additional benefit of not just reproducing a given
+scenario, but transferring software use to new problems, domains, and/or
+applications, justifies developing research software with a long-term
+perspective as *sustainable research software*.
+
+In order to support research, a sustainable software must be
+correct (Hatton 2007; Chang et al. 2006; Matthews 2007), validatable,
+understandable, documented, publicly released, adequately published
+(i.e. in persistently identifiable form as software source code (Smith
+et al. 2016), and potentially in an additional paper which describes the
+software concept, design decisions and development rationale), actively
+maintained, and  (Merali 2010; Barnes 2010; Tse 2010).
+
+To make it possible to reproduce computationally supported research, a
+software that has been used must continue to exist in the first place,
+but also continue to be usable, understandable, and reproducible in the
+evolving software and hardware environment. Moreover, the software
+should support reuse scenarios to avoid duplication of efforts, and
+unneeded drain of resources. If it is publicly funded, it should also be
+open source and freely accessible.
+
+Currently, creating and using sustainable research software is not yet
+sufficiently incentivized. To evaluate where to address this
+shortcoming, we have identified the following challenges:
+
+-   **Lack of benefit for the individual**: The primary motivation for
+    sustainable research software is the common benefit, rather than the
+    individual benefit. While directing resources towards sustainable
+    research software may in the short run yield disadvantages for the
+    developing individual (e.g., fewer paper publications) (Bangerth and
+    Heister 2014; Prins et al. 2015), it is clearly beneficial for the
+    research community as a whole, as it enables better and more
+    research by freeing funds for actual research rather than
+    (repetitive) software development.
+
+-   **Lack of awareness**: Research software sustainability
+    (cf. (Venters et al. 2014; Goble 2014; Druskat 2016; Katz 2018)) and
```

Comment:
```suggestion
    (cf. Venters et al. 2014; Goble 2014; Druskat 2016; Katz 2018) and
```

What does `cf.` stand for?

Reply:
> short for the Latin: confer/conferatur, both meaning "compare"
([Wikipedia](https://en.wikipedia.org/wiki/Cf.))

It also confuses me often, it is not as common as "e.g.", "i.e.", which are already confusing by themselves.

Reply:
The brackets are an artifact of TeX to Markdown conversion. Could change cf. to "see" in the last editorial round.